0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py

```diff
@@ -0,0 +1,109 @@
+""" This file provides analysis utilities for a variety of tasks """
```

Comment:
why is this file saved in `nbconverted` folder? Is it originally a `.ipynb` file?

I typically expect to only see nbconverted files in here, not utility scripts (they typically go in a `utils` or `scripts` folder

Reply:
Good catch, forgot to add the file to a utils directory